strings.txt

```diff
@@ -16858,3 +16858,955 @@
+    en = Carports
+    ru = Капитальный навес
+    fr = Abris auto
+    da = Carporte
+    id = Parkir Beratap
+    ko = 간이 차고
+    sv = Garage
+    tr = Açık Park Alanları
+    uk = Капітальний навіс
+    vi = Nhà để xe
+    hu = Nagyobb fedett parkoló
+    de = Parkhalle
+    fi = Autokatokset
+    cs = Přístřešek
+    it = Tettoia
+    nb = Carport
+    zh-Hans = 屋侧车库
+    th = โรงรถ
+    ja = カーポート
+    ro = Garaje
+    ar = مظلات سيارات
+    sk = Otvorené garáže
+    es = Cochera
+    pl = Zadaszenia dla samochodów
+    nl = Carports
+
+  [placepage_parking_boxes]
+    en = Garage boxes
+    ru = Гаражный бокс
+    fr = Box de garage
+    da = Garagekasser
+    id = Sasana tinju
+    ko = 차고
+    sv = Garageboxar
+    tr = Garaj kutuları
+    uk = Гаражний бокс
+    vi = Thùng gara
+    hu = Garázsbox
+    de = Garagenbox
+    fi = Autotalliruudut
+    cs = Garáž
+    it = Box
+    nb = Garasjebokser
+    zh-Hans = 停车房
+    th = โรงรถแบบกล่อง
+    ja = ガレージボックス
+    ar = صناديق جراج
+    sk = Hromadné garáže
+    es = Garaje
+    pl = Miejsca parkingowe w garażu
+    nl = Garageboxen
+
+  [placepage_parking_pay]
+    en = Pay
+    ru = Платная
+    fr = Payer
+    da = Betal
+    id = Berbayar
+    ko = 유료
+    sv = Kostnadsbelagd
+    tr = Ücretli
+    uk = Платне
+    vi = Trả phí
+    hu = Fizetős
+    de = Gebührenpflichtig
+    fi = Maksullinen
+    cs = Placené
+    it = A pagamento
+    nb = Betaling
+    zh-Hans = 付款
+    th = เสียเงิน
+    ja = 支払い
+    ro = Cu plată
+    ar = دفع
+    sk = Platené
+    es = De pago
+    pl = Płatny
+    nl = Betalen
+
+  [placepage_parking_free]
+    en = Free
+    ru = Бесплатная
+    fr = Gratuit
+    da = Gratis
+    id = Gratis
+    ko = 무료
+    sv = Gratis
+    tr = Ücretsiz
+    uk = Безкоштовне
+    vi = Miễn phí
+    hu = Díjmentes
+    de = Gebührenfrei
+    fi = Ilmainen
+    cs = Zdarma
+    it = Gratis
+    nb = Gratis
+    zh-Hans = 免费
+    th = ฟรี
+    ja = 無料
+    ro = Gratuit
+    ar = مجانًا
+    sk = Zdarma
+    es = Gratuito
+    pl = Darmowy
+    nl = Gratis
+
+  [placepage_parking_interval]
+    en = Interval Pay
+    ru = Переменно платная
+    fr = Période de paiement
+    da = Intervalbetaling
+    id = Biaya Rutin
+    ko = 주차요금 간격
+    sv = Delvis kostnadsbelagd
+    tr = Süre Ücretli
+    uk = Змінно платне
+    vi = Trả Phí Cách Quãng
+    hu = Idősávos fizetős
+    de = Teilweise gebührenpflichtig
+    fi = Aikamaksu
+    cs = Placeno za čas
+    it = Pagamento a tempo
+    nb = Intervallbetaling
+    zh-Hans = 间隔付款
+    th = จ่ายตามช่วงเวลา
+    ja = 支払間隔
+    ro = Interval de plată
+    ar = دفع عن فترة
+    sk = Platba v intervaloch
+    es = Pago por intervalos
+    pl = Płatny na godziny
+    nl = Intervalbetaling
+
+  [placepage_entrance_number]
+    comment = Entrance number in Place Page
+    en = No.
+    ru = №
+    fr = N°
+    da = Nr.
+    id = Nomor
+    ko = 번호
+    sv = Nr
+    tr = No.
+    uk = №
+    vi = Số
+    hu = Szám
+    de = Nr. der Einfahrt
```

Comment:
Эээм, не уверен. Наташа нужна.
> On Apr 26, 2016, at 4:39 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61086249>:
> 
> > +    nl = Intervalbetaling
> > +
> > +  [placepage_entrance_number]
> > +    comment = Entrance number in Place Page
> > +    en = No.
> > +    ru = №
> > +    fr = N°
> > +    da = Nr.
> > +    id = Nomor
> > +    ko = 번호
> > +    sv = Nr
> > +    tr = No.
> > +    uk = №
> > +    vi = Số
> > +    hu = Szám
> > +    de = Nr. der Einfahrt
> Просто Nr.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295..454ca48abdc1a7a643a178fac869c249f100121f#r61086249>